Sources/Workspace/Workspace+Dependencies.swift

```diff
@@ -966,6 +961,9 @@ extension Workspace {
         /// The package is updated.
         case updated(State)
 
+        /// The package is replaced with a prebuilt library
+        case usesLibrary(ProvidedLibrary)
```

Comment:
@MaxDesiatov I was going back and forth on this - it would be possible to adjust `unchanged` to optionally carry this and then if version has a provided library the state would always be `unchanged`. WDYT?

Reply:
I don't have a strong opinion, but for some reason I slightly prefer the non-optional case like it is now.